src/components/CopyBox/CopyBox.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Form from "~/components/Form";
+import Button from "~/components/Button";
+
+const CopyBox = ({ value }) => {
```

Comment:
This component creates a box with an icon which when clicked copies the text into a clipboard.